FEXCore/Source/Interface/Core/JIT/MemoryOps.cpp

```diff
@@ -1628,13 +1623,14 @@ DEF_OP(StoreMemPredicate) {
   }
 }
 
-DEF_OP(LoadMemPredicate) {
-  const auto Op = IROp->C<IR::IROp_LoadMemPredicate>();
+DEF_OP(LoadMemX87SVEOptPredicate) {
+  const auto Op = IROp->C<IR::IROp_LoadMemX87SVEOptPredicate>();
   const auto Dst = GetVReg(Node);
-  const auto Predicate = GetPReg(Op->Mask.ID());
+  const auto Predicate = PRED_X87_SVEOPT;
   const auto MemReg = GetReg(Op->Addr.ID());
 
-  LOGMAN_THROW_A_FMT(HostSupportsSVE128 || HostSupportsSVE256, "LoadMemPredicate needs SVE support");
+  LOGMAN_THROW_A_FMT(HostSupportsSVE128 || HostSupportsSVE256, "LoadMemX87SVEOptPredicate needs SVE support");
+  ptrue(ARMEmitter::SubRegSize::i16Bit, Predicate, ARMEmitter::PredicatePattern::SVE_VL5);
```

Comment:
I think you need to remove this otherwise the patch will do nothing :) (which is fine as it'll always be set in FillSRA)

Reply:
*sigh* of course.